subsys/bluetooth/controller/include/ll.h

```diff
@@ -53,7 +53,8 @@ uint8_t ll_adv_enable(uint8_t handle, uint8_t enable,
 		   uint8_t at_anchor, uint32_t ticks_anchor, uint8_t retry,
 		   uint8_t scan_window, uint8_t scan_delay);
 #else /* !CONFIG_BT_HCI_MESH_EXT */
-uint8_t ll_adv_enable(uint8_t handle, uint8_t enable);
+uint8_t ll_adv_enable(uint8_t handle, uint8_t enable,
```

Comment:
Please fix the babblesim tests: 
the calls in tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c needs to be updated so that the simulator tests can run

Reply:
Fixed